integreat_cms/api/urls.py

```diff
@@ -129,10 +138,91 @@
     path("<slug:region_slug>/", region_by_slug, name="region_by_slug"),
 ]
 
+social_media_prefix = "social"
+
+social_media_api_urlpatterns = [
+    path("", root_social_media_headers, name="social_root"),
+    *map(
+        lambda reserved: path(
+            f"{reserved}/",
+            include(
+                [
+                    path(
+                        "",
+                        root_social_media_headers,
+                        name=f"social_root_reserved_{reserved}",
+                    ),
+                    path(
+                        "<slug:language_slug>/",
+                        root_social_media_headers,
+                        name=f"social_root_reserved_default_language_{reserved}",
+                    ),
+                ]
+            ),
+        ),
+        settings.RESERVED_REGION_SLUGS,
+    ),
+    path(
+        "<slug:social_region_slug>", region_social_media_headers, name="social_region"
+    ),
+    path(
+        "<slug:region_slug>/",
+        region_social_media_headers,
+        name="social_region_default_language",
+    ),
+    path(
+        "<slug:region_slug>/<slug:language_slug>/",
+        include(
+            [
+                path("", region_social_media_headers, name="social_region"),
+                path(
+                    "news/local/",
+                    region_social_media_headers,
+                    name="social_region_reserved_local_news",
+                ),
+                re_path(
+                    r"^news/tunews/[\w-]+/$",
+                    region_social_media_headers,
+                    name="social_region_reserved_tunews",
+                ),
+                *map(
+                    lambda reserved: path(
+                        reserved,
```

Comment:
```suggestion
                        f"{reserved}/",
```
Just for consistency, I'd also use a trailing slash here.